js/script.js

```diff
@@ -29,3 +29,21 @@ contact.addEventListener("click", () => {
 const close_menu = () => {
     menuBody.classList.toggle("invisible");
 };
+
```

Comment:
- [ ] It would be better to set correctly [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in your project. 😉